distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.4.6 (in development)
 ------------------------------------------------------------------------
+- Feature: [Plugin] Added API for returning metadata from all registered plugins.
```

Comment:
We typically have the entry for plugins like this.
`- Feature: [#18675] [Plugin] Plugins can refer to g2 image icons by name.`
You can use your PR number

Reply:
Got it, I added the PR number and re-worded it a bit